src/mmw/js/src/modeling/gwlfe/subbasin/views.js

```diff
@@ -224,6 +226,85 @@ var Huc12TotalsTableView = Marionette.ItemView.extend({
     }
 });
 
+var CatchmentsTableView = Marionette.ItemView.extend({
+    template: catchmentTableTmpl,
+    ui: {
+        'rows': '.subbasin-catchment-row',
+    },
+    events: {
+        'mouseover @ui.rows': 'handleRowMouseOver',
+        'mouseout @ui.rows': 'handleRowMouseOut',
+    },
+    onRender: function() {
+        $('[data-toggle="table"]').bootstrapTable();
+    },
+    initialize: function() {
+        var self = this;
+        self.catchmentDetails = App.currentProject.get('subbasins').getActive().get('catchments');
+        if (this.catchmentDetails.isEmpty()) {
+            self.listenToOnce(self.catchmentDetails, 'add', this.setupCatchmentDetails, this);
+        } else {
+            this.setupCatchmentDetails();
```

Comment:
Calling `render` within `initialize` feels like a mistake ... but I'm not seeing any errors in the console. It is possible that this is the cause of the broken bootstrap table. Consider moving it to a custom function and calling that directly here, and also from within `onRender`.